test/Microsoft.Azure.Devices.Edge.Test/helpers/TestResultCoordinatorUtil.cs

```diff
@@ -85,7 +85,16 @@ public static async Task ValidateResultsAsync()
             HttpClient client = new HttpClient();
             HttpResponseMessage response = await client.GetAsync("http://localhost:5001/api/report");
             var jsonstring = await response.Content.ReadAsStringAsync();
-            bool isPassed = (bool)JArray.Parse(jsonstring)[0]["IsPassed"];
+            bool isPassed;
```

Comment:
Should we be checking the status code? Since a non-200 code was returned, a JSON response isn't guaranteed and we shouldn't try to parse it as JSON (although I suppose a try/catch is still warranted).

Reply:
I think it doesn't really matter as long as it spits out the contents for a dev to debug with. The important part is getting the actual error instead of just "Could not parse output"